test/colony.js

```diff
@@ -664,34 +687,22 @@ contract("Colony", accounts => {
         })
       );
 
-      const evaluatorInfo = await colony.getTaskRole(taskId, EVALUATOR_ROLE);
-      assert.equal(evaluatorInfo[0], "0x0000000000000000000000000000000000000000");
-
       const workerInfo = await colony.getTaskRole(taskId, WORKER_ROLE);
       assert.equal(workerInfo[0], "0x0000000000000000000000000000000000000000");
     });
 
     it("should be able to remove evaluator role", async () => {
       const taskId = await makeTask({ colony });
 
-      await executeSignedRoleAssignment({
-        colony,
-        taskId,
-        functionName: "setTaskEvaluatorRole",
-        signers: [MANAGER, EVALUATOR],
-        sigTypes: [0, 0],
-        args: [taskId, EVALUATOR]
-      });
-
       let evaluator = await colony.getTaskRole(taskId, EVALUATOR_ROLE);
-      assert.equal(evaluator[0], EVALUATOR);
+      assert.equal(evaluator[0], MANAGER);
```

Comment:
We should really leave `EVALUATOR` here. Any logic that checks that the evaluator role is in fact assigned to the manager may lead to misunderstandings

Reply:
Ok, but I'd like to add a note then that only the manager's signature is required on the unassignment because `manager === evaluator`